src/plugins/data/public/mocks.ts

```diff
@@ -26,20 +27,7 @@ const createSetupContract = (): Setup => {
 
 const createStartContract = (): Start => {
   const queryStartMock = queryServiceMock.createStartContract();
-  const dataViews = {
-    find: jest.fn((search) => [{ id: search, title: search }]),
-    createField: jest.fn(() => {}),
-    createFieldList: jest.fn(() => []),
-    ensureDefaultIndexPattern: jest.fn(),
-    make: () => ({
-      fieldsFetcher: {
-        fetchForWildcard: jest.fn(),
-      },
-    }),
-    get: jest.fn().mockReturnValue(Promise.resolve({})),
-    clearCache: jest.fn(),
-    getIdsWithTitle: jest.fn(),
-  } as unknown as DataViewsContract;
+  const dataViews = dataViewPluginMocks.createStartContract();
```

Comment:
The `dataViewPluginMock` is more complete and includes things that are necessary for the new Dashboard service abstraction, so using that instead.